4001-5000/LIT4615Salam.xml

```diff
@@ -50,17 +50,32 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         </profileDesc>
         <revisionDesc>
             <change who="MV" when="2017-04-06">Created file</change>
+            <change who="DR" when="2020-06-11">Added relation and explicit</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT4615Salam" passive="LIT1968Mashaf"/>
+                </listRelation>
+            </div>
             <div type="edition">
                 <div type="textpart" subtype="incipit" xml:lang="gez">
                     <ab>
                         ለመስቀልከ፡ እግዚኦ፡ ንፌኑ፡ ስብሐተ፡ ሃሌሉያ፡ ሰላም፡ ለከ፡ ዮሐንስ፡ ወልደ፡ ነጐድጓድ፨ 
                         ሰላም፡ ለከ፡ ዮሐንስ፡ አቡ፡ ቀለምሲስ፨ ሰላም፡ ለከ፡ ዮሐንስ፡ ቴዎጎሎስ፨
                     </ab>
                 </div>
+                
+                <div type="textpart" subtype="explicit" xml:lang="gez">
+                    <note xml:lang="en">The following explicit is taken from <ref type="mss" corresp="BDLaethd20"/></note>
+                    <ab>ወላዲተ፡ አምላክ፡ ማርያም፡ እንበለ፡ ሰብሳብ፡ ወሩካቤ፡ በል፨
+                        ወእምዝ፡ ሰላም፡ ለኪ፡ ማርያም፡ ኆኅተ፡ ምሥራቅ፡ ዘወልደ፡ ኖሬ፡ ሰላም፡ ለኪ፡ ዓፀድ፡ ዘኤስኬድሬ፨
+                        ሰላም፡ ለኪ፡ ሥርጉት፡ በወራውሬ፨ ሰ፡ ለ፡ እንተ፡ ወለድኪ፡ ፈጣሬ፨
```

Comment:
We may state that ሰ፡ ለ፡ is the abbreviated form of ሰላም፡ ለኪ፡ (e.g. through `<expan><abbr></abbr><ex></ex>`?)